website/content/docs/k8s/installation/vault/data-integration/webhook-certs.mdx

```diff
@@ -0,0 +1,261 @@
+---
+layout: docs
+page_title: Vault as the Webhook Certificate Provider for Consul Controller and Connect Inject on Kubernetes
+description: >-
+  Configuring the Consul Helm chart to use TLS certificates issued by Vault for the Consul Controller and Connect Inject webhooks.
+---
+
+# Vault as the Controller and Connect Inject Webhook Certificate Provider on Kubernetes
+## Prerequisites
```

Comment:
```suggestion
# Vault as the Controller and Connect Inject Webhook Certificate Provider on Kubernetes

This topic describes how to configure the Consul Helm chart to use TLS certificates issued by Vault in the Consul controller and connect inject webhooks.

## Prerequisites
```
Per the style guide, we don't want to introduce sections so that headings aren't stacked on top of each other.